### ISSUE TITLE
high keypoint loss function

### ISSUE THREAD

**Issue by PalashJauhari** [FangYang970206/MaskRCNN-Keypoint-Demo#3]
hello, I can see your question here:

https://github.com/chrispolo/Keypoints-of-humanpose-with-Mask-R-CNN/issues/5

were you able to solve this problem? or high loss function is correct?

**Comment by FangYang970206** [comment]
Not completely resolved, the loss of the second version of the pre-training model( commits "update model download links (more powerful)" ) has become smaller.
